tests/spec_tests/type_constraint_tests/README.md

```diff
@@ -59,10 +59,10 @@ Here is an example of such a test, followed by a brief explanation:
 ```
 In this example, the query contains the expression `arg1 + arg2`, an addition operation.
 There is one map in the `valid_types` sequence. It indicates that `arg1` can be any
-numeric type, `NULL`, or missing; it also indicates that `arg2` can be any numeric type,
-`NULL`, or missing. Since these lists are included in the same map, that means any
```

Comment:
Patrick recently realized we can't actually support `MISSING` in these tests using the test runner as implemented because it is invalid for a field's sole schema to be `MISSING`. So we removed missing from the test suite.

This change is just some lingering clean up to remove references to missing.